vtrace/envitools.py

```diff
@@ -25,36 +36,134 @@ def cmpRegs(emu, trace):
     return True
 
 
-def emuFromTrace(trace):
-    '''
-    Produce an envi emulator for this tracer object.
-    '''
-    arch = trace.getMeta('Architecture')
-    plat = trace.getMeta('Platform')
-    amod = envi.getArchModule(arch)
-    emu = amod.getEmulator()
+skip_mem_opcodes_by_arch = {
+        'i386': (e_i386const.INS_NOP, e_i386const.INS_LEA),
+        'amd64': (e_i386const.INS_NOP, e_i386const.INS_LEA),
```

Comment:
Should we also add in support for ARM/etc?

Reply:
i don't know of any offhand for other architectures.  we can certainly add them as we come across them.